src/main/target/CFGF4OMNI/target.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USE_TARGET_CONFIG
+
+#define TARGET_BOARD_IDENTIFIER "F4OM"
+#define USBD_PRODUCT_STRING "Betaflight F4 Omni"
+
+#define LED0_PIN                PB5
+#define BEEPER                  PB4
+#define BEEPER_INVERTED
+
+#define USE_DSHOT_DMAR
+
+// These inverter control pins collide with timer channels on CH5 and CH6 pads.
+// Users of these timers/pads must un-map the inverter assignment explicitly.
+#define INVERTER_PIN_UART6      NONE // PC8 // Omnibus F4 V3 and later
+#define INVERTER_PIN_UART3      NONE // PC9 // Omnibus F4 Pro Corners
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+
+#define MPU6000_CS_PIN          PA4
+#define MPU6000_SPI_INSTANCE    SPI1
+
+// MPU6000 interrupts
+#define USE_EXTI
+#define MPU_INT_EXTI            PC4
```

Comment:
How does a user reconfigure this pin?

Reply:
(Re)configurability around acc/gyro is still very weak.

Reply:
Oh, and yes, per driver EXTI is not supported at all atm.